lib/openai/client.rb

```diff
@@ -54,5 +54,32 @@ def transcribe(parameters: {})
     def translate(parameters: {})
       OpenAI::Client.multipart_post(path: "/audio/translations", parameters: parameters)
     end
+
+    def token_count(parameters: {})
+      contents = parameters[:messages]&.map { |message| message[:content] }&.join(" ") || nil
```

Comment:
I'd like to vote against forcing this method to expect multiple messages by default. Consider using a single `content = ""` parameter and reducing the complexity here.

Reply:
Agree with this